src/OSPSuite.Core/Commands/Core/Command.cs

```diff
@@ -93,7 +93,7 @@ public interface IReversibleCommand<in TExecutionContext> : ICommand<TExecutionC
       /// <summary>
       ///   Create an inverse for the current command
       /// </summary>
-      IReversibleCommand<TExecutionContext> InverseCommand(TExecutionContext context);
+      ICommand<TExecutionContext> InverseCommand(TExecutionContext context);
```

Comment:
@Yuri05 I am changing this to not require the undo command to also be undoable. That way we can have the best of both worlds